docker/tasks/main.yml

```diff
@@ -38,4 +41,3 @@
       - docker-ce-cli
       - containerd.io
     update_cache: yes
```

Comment:
Since you're doing this in the previous task, no reason to update_cache yet again, right?

Reply:
Removed the update_cache.